quickstack/manifests/controller_common.pp

```diff
@@ -893,7 +893,8 @@
          "puppet:///modules/sensu/plugins/cpu-pcnt-usage-metrics.rb",
          "puppet:///modules/sensu/plugins/disk-metrics.rb",
          "puppet:///modules/sensu/plugins/vmstat-metrics.rb",
-         "puppet:///modules/sensu/plugins/iostat-metrics.rb"
+         "puppet:///modules/sensu/plugins/iostat-metrics.rb",
+         "puppet:///modules/sensu/plugins/vm-metrics.py"
```

Comment:
We don't create instances from the controller node...I guess this should be considered.